config.json

```diff
@@ -1368,6 +1368,21 @@
         "strings"
       ]
     },
+    {
+      "slug": "affine-cipher",
+      "uuid": "02bf6783-fc74-47e9-854f-44d22eb1b6f8",
+      "core": false,
+      "unlocked_by": "grade-school",
+      "difficulty": 5,
+      "topics": [
+        "algorithms",
+        "cryptography",
+        "exception_handling",
+        "loops",
+        "math",
+        "strings"
```

Comment:
`topics` refers only to those concepts that are integral to the problem, and not to a potential solution. In this case, `exception_handling` (see `error-handling` exercise), `loops`, and `math` (reserved for exercises that deal with advanced mathematical concepts, not just any exercise that contains a single mathematical operation) should be left out.